README.md

```diff
@@ -1,2 +1,180 @@
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 <a name="about-project"></a>
+
+> We built an app for a group of people, we call anime enthusiasts to watch animes information, details and many more using MAL API. The application allows the users to "add to list" anime and explore anime world with informations and queries.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Anime Information: Explore a vast collection of anime titles, including popular and trending shows. Get detailed information about each anime, such as synopsis, genres, episode count, airing status, and more.
+
+- Anime Details: Dive deeper into your favorite anime series. Access episode lists, character profiles, voice actors, ratings, and reviews. Stay up-to-date with the latest news and announcements related to your favorite shows.
+
+- Add to List: Create your personal anime watchlist by adding shows to your list. Keep track of the anime you plan to watch, are currently watching, or have completed. Mark episodes as watched to keep your progress updated.
+
+- Genre Exploration: Discover anime based on specific genres of your interest. Browse through various genres such as action, romance, comedy, fantasy, and more. Find hidden gems and explore new genres to broaden your anime horizons.
+
+- Search Functionality: Easily search for anime titles, characters, or specific episodes. Use keywords, filters, or advanced search options to quickly find what you're looking for. Save your favorite searches for future reference.
+
+- User Profiles: Create a personalized user profile to showcase your anime preferences and watch history. Connect with other anime enthusiasts, share recommendations, and discuss your favorite shows in a vibrant community
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
```

Comment:
- [ ] Dear @MoyasiGinko, Just as mentioned by the previous reviewer, Please deploy your application using Render, Netlify, or GitHub Pages, after confirming the home and detail page works as expected, add the live demo linked to your readme.